src/pages/Signup/Signup.js

```diff
@@ -19,150 +19,79 @@ class Signup extends React.Component {
     });
   };
 
-  // Validation을 체크하는 함수 (구현중...)
-  // doValidation = e => {
-  //   const userInputs = Object.entries(this.state);
-  //   console.log(userInputs);
-  //   userInputs.forEach(el => {
-  //     console.log(validationFor[el[0]](el[1]));
-  //     if (!validationFor[el[0]](el[1])) {
-  //       return alert('양식에 맞지 않습니다');
-  //     }
-  // });
-
-  // 리팩토링 이전 Validation 함수
-  // if (!validationName(this.state.name)) {
-  //   alert('이름을 입력해주세요.');
-  // }
-  // if (!validationMobile(this.state.mobile)) {
-  //   alert('번호를 입력해주세요.');
-  // }
-  // if (!validationEmail(this.state.email)) {
-  //   alert('이메일 형식이 유효하지 않습니다.');
-  // }
-  // if (!validationPwd(this.state.pw)) {
-  //   alert('영문,숫자,특수문자를 혼합하여 6~20자 이내로 입력해주세요.');
-  // }
-  // if (
-  //   validationName(this.state.name) &&
-  //   validationMobile(this.state.mobile) &&
-  //   validationEmail(this.state.email) &&
-  //   validationPwd(this.state.pw)
-  // ) {
-  //   return true;
-  //   // alert('회원가입 완료');
-  // } else {
-  //   alert('입력란을 다시 확인하세요.');
-  //   return false;
-  // }
-  // };
+  doValidation = () => {
+    const userInputs = Object.entries(this.state);
+    userInputs.forEach(el => {
+      if (!validationFor[el[0]](el[1])) {
+        return alert('양식에 맞지 않습니다');
+      }
+    });
+    return this.requestSignup();
+  };
 
   // Back이랑 연결하는 fetch 함수
   requestSignup = () => {
+    const { name, mobile, email, password } = this.state;
     fetch(`${POST_SIGNUP_API}`, {
       method: 'POST',
       body: JSON.stringify({
-        name: this.state.name,
-        mobile: this.state.mobile,
-        email: this.state.email,
-        password: this.state.pw,
+        name: name,
+        mobile: mobile,
+        email: email,
+        password: password,
```

Comment:
객체 단축 속성명 이용할 수 있습니다.
```suggestion
name, mobile, email, password
```

Reply:
음.... 정확이 이해를 못하겠어요 ㅠㅠ

Reply:
수정했습니다.